src/SFA.DAS.AssessorService.Application.Api/Controllers/OrganisationStandardController.cs

```diff
@@ -156,5 +155,20 @@ public async Task<IActionResult> AddOrganisationStandard([FromBody] Organisation
                 return BadRequest(new EpaoStandardVersionResponse(ex.Message));
             }
         }
+
+        [HttpPost("organisationstandard/withdraw", Name = "WithdrawStandard")]
+        [ValidateBadRequest]
+        [SwaggerResponse((int)HttpStatusCode.NoContent)]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, Type = typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> WithdrawalStandard(
```

Comment:
Perhaps you meant to call it `WithdrawStandard` or `WithdrawFromStandard`?

Reply:
Done